mmv1/products/iamworkforcepool/WorkforcePoolProvider.yaml

```
@@ -70,6 +70,16 @@ examples:
       org_id: :ORG_ID
     ignore_read_extra:
       - 'oidc.0.client_secret.0.value.0.plain_text'
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'iam_workforce_pool_provider_oidc_upload_key'
+    min_version: beta
+    skip_docs: true
+    primary_resource_id: 'example'
+    vars:
+      workforce_pool_id: 'example-pool'
+      provider_id: 'example-prvdr'
+    test_env_vars:
+      org_id: :ORG_ID
```

It looks like these tests need to ignore the value for `oidc.client_secret.value.plain_text` during ImportStateVerify because it's not returned by the API.

```suggestion
      org_id: :ORG_ID
    ignore_read_extra:
      - 'oidc.0.client_secret.0.value.0.plain_text'
```